test/src/test/java/org/corfudb/integration/WorkflowIT.java

```diff
@@ -399,4 +398,27 @@ public void addNodeWithTrim() throws Exception {
         // Shutdown two nodes
         run(n1.shutdown, n2.shutdown);
     }
+
+    /**
+     * A Helper method which created a Corfu Runtime for a particular node. The created runtime uses the corresponding
+     * properties of accompanied instance of Harness used for creating the cluster
+     *
+     * @param node
+     * @param harness
+     *
+     * @return an instance of CorfuRuntime configured with TLS parameters provided in CorfuDB.properties
+     */
+    private CorfuRuntime createRuntimeForHarnessAt(Node node, Harness harness) {
```

Comment:
This is a utility method. Can we move it to the Harness class?

Reply:
Good point, I think it makes the harness more close to idea of making the harness the central point for creating the testbeds like we talked about.
Done.